spec.html

```diff
@@ -5836,7 +5836,7 @@ <h1>
         1. If ! ToString(_n_) is _argument_, return _n_.
         1. Return *undefined*.
       </emu-alg>
-      <p>A <dfn variants="canonical numeric strings">canonical numeric string</dfn> is any String value for which the CanonicalNumericIndexString abstract operation does not return *undefined*.</p>
+      <p>A <dfn variants="canonical numeric strings">canonical numeric string</dfn> is any String value for which the CanonicalNumericIndexString abstract operation does not return *undefined*. Each such String value corresponds with a distinct Number value.</p>
```

Comment:
```suggestion
      <p>A <dfn variants="canonical numeric strings">canonical numeric string</dfn> is any String for which the CanonicalNumericIndexString abstract operation does not return *undefined*. Each such String corresponds with a distinct Number.</p>
```

We don't say "X value" when X alone would work anymore. Though I'm struggling to find where that precedent was set, and we don't seem to have it in our editorial conventions doc.

Reply:
Done.